### PR TITLE
Add LL/DDI/ADV/BI-05-C and LL/DDI/ADV/BI-06-C test cases

### PR DESCRIPTION
**Pull request by Tronil**
Added the two (very similar) simple tests

Signed-off-by: Troels Nilsson <trnn@demant.com>

### CODE REVIEW

**Comment by Tronil** [comment]
Just noticed that I've used the wrong HCI command for LL/DDI/ADV/BI-06-C (it should of course have been le_set_extended_scan_response_data for that case instead of le_set_extended_advertising_data) - I'll quickly fix that up

File: src/tests/ll_verification.py [hunk 1]
```diff
@@ -3781,6 +3781,72 @@ def ll_con_per_bv_81_c(transport, upperTester, lowerTester, trace):
 
     return success;
 
+# Implements LL/DDI/ADV/BI-05-C and LL/DDI/ADV/BI-06-C test cases (only difference is the event properties)
```
Comment (wopu-ot) [inline src/tests/ll_verification.py:3784]: I think these test cases should be placed next to the other LL/DDI/ADV test cases to not make this file messier than it already is.

**Review by wopu-ot — CHANGES_REQUESTED** [review]
Please move the new test cases next to the existing LL/DDI/ADV test cases, otherwise LGTM

File: src/tests/ll_verification.py (hunk 1 above)
Comment (Tronil) [reply src/tests/ll_verification.py]: Moved

**Review by wopu-ot — APPROVED** [review]

**Pull request merged**